sim/team-validator.ts

```diff
@@ -1964,7 +1968,7 @@ export class TeamValidator {
 						if (dex.gen >= 5 && learnedGen <= 4 && [
 							'cut', 'fly', 'surf', 'strength', 'rocksmash', 'waterfall', 'rockclimb',
 						].includes(moveid)) {
-							cantLearnReason = `can't be transferred from Gen 3 to 4 because it's an HM move.`;
+							cantLearnReason = `can't be transferred from Gen 4 to 5 because it's an HM move.`;
```

Comment:
This seems unrelated to the rest of the PR.

Reply:
It is, but it is also something that needs to be fixed (appears to have been copy/pasted from the 3 -> 4 check above). Since I'm editing the function already, I don't see a reason to create a whole new PR for a single line.

Reply:
@KrisXV awaiting your opinion on this please

Reply:
bump